### PR TITLE
Soop reduce ship callsign calls

### PR DESCRIPTION
**Pull request by lbesnard**

### CODE REVIEW

File: test_aodndata/soop/test_soop_ba.py [hunk 1]
```diff
@@ -37,6 +39,7 @@ def setUp(self):
         self.handler_class = SoopBaHandler
         super(TestSoopBaHandler, self).setUp()
 
+    @patch("aodndata.soop.soop_ba.SHIP_CALLSIGN_LS", SHIP_CALLSIGN_LS)
```
Comment (bpasquer) [inline test_aodndata/soop/test_soop_ba.py]: Can't you add the patch once and for all at the beginning of the class definition?

**Comment by lbesnard** [comment]
@lwgordonimos wouldn t mind your review on this

**Comment by lbesnard** [comment]
@lwgordonimos done !

**Comment by lbesnard** [comment]
@lwgordonimos could we merge this one ?

**Pull request merged**